### ISSUE TITLE
How to reproduce the results of GlobalTemp dataset?

### ISSUE THREAD

**Issue by zezhishao** [Time-MoE/Time-MoE#44]
Hi,

First of all, thank you for your outstanding work, and congratulations on your paper being accepted at ICLR!

I noticed that the reported results of GlobalTemp in your paper appear to be normalized. Could you clarify how the data was normalized and which portion of the data was used for metric calculation? Additionally, I would appreciate any guidance on how to reproduce the results on GlobalTemp.

Looking forward to your insights!

**Comment by Maple728** [comment]
Thank you for your interest in our work!

The `GlobalTemp` dataset is sourced from [CorrFormer](https://github.com/thuml/Corrformer). We select the first 1000 dimensions and train a ZeroScaler using the training dataset for each dimension, which is then applied to the test dataset. All 1000 dimensions are used for metric calculation.

To evaluate Time-MoE, applying it on the scaled test dataset directly yields the zero-shot result. For the full-shot result, first fine-tune Time-MoE on the prepared training dataset and then evaluate it on the scaled test dataset.

Please fell free to contact us, if you have any questions.

**Comment by MogicianXD** [comment]
Could you kindly upload the dataset to your Google drive? It would be much convenient for reproduction. Thanks!

**Comment by Maple728** [comment]
> Could you kindly upload the dataset to your Google drive? It would be much more convenient for reproduction. Thanks!

Sure, I will upload the Global Temp dataset used by Time-MoE shortly. After that, I will add the Google Drive link to the README file.

**Comment by Maple728** [comment]
The Global Temperature datasets can be accessed via [train](https://drive.google.com/drive/folders/1CsigJ67aje3ei1JG_qKWxIHxLM0blEiH?usp=sharing) and [test](https://drive.google.com/drive/folders/1KjnAYr9X3D-jyJpo4yM7Giyq5V1Hga_7?usp=sharing) links.

**Comment by MogicianXD** [comment]
Many thanks!
